data/fajnzylber2020sars/fajnzylber2020sars.yaml

```diff
@@ -0,0 +1,1733 @@
+title: SARS-CoV-2 viral load is associated with increased disease severity and mortality
+doi: 10.1038/s41467-020-19057-5
+description: >
+  The paper quantified SARS-CoV-2 viral load from participants with a diverse range
+  of COVID-19 disease severity, including those requiring hospitalization, outpatients
+  with mild disease, and individuals with resolved infections. Nasopharyngeal swabs,
+  oropharyngeal swabs, sputum, and urine were collected from hospitalized participants.
+  Data were obtained from the supplementary materials.
+analytes:
+  Nasopharyngeal_SARSCoV2_N:
+    description: >
+      Nasopharyngeal swabs were collected in 3 mL of phosphate-buffered saline. Viral
+      concentrations were quantified using RT-qPCR targeting the N gene in the nasopharyngeal
+      swab samples.
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: nasopharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/mL
+    reference_event: symptom onset
+  Oropharyngeal_PBS_SARSCoV2_N:
+    description: >
+      Oropharyngeal swabs were collected in 3 mL of phosphate buffered saline. Viral
+      concentrations were quantified using RT-qPCR targeting the N gene in oropharyngeal
+      swab samples.
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: oropharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/mL
+    reference_event: symptom onset
+  Oropharyngeal_VTM_SARSCoV2_N:
+    description: >
+      Oropharyngeal swabs were collected in viral transport medium. Viral concentrations
+      were quantified using RT-qPCR targeting the N gene in oropharyngeal swab samples.
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: oropharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/mL
+    reference_event: symptom onset
+  Sputum_SARSCoV2_N:
+    description: >
+      Sputum samples were collected and viral concentrations were quantified using
+      RT-qPCR targeting the N gene.
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: sputum
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/mL
+    reference_event: symptom onset
+  Plasma_SARSCoV2_N:
+    description: >
+      Plasma samples were collected and viral concentrations were quantified using
+      RT-qPCR targeting the N gene.
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: plasma
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/mL
+    reference_event: symptom onset
+  Urine_SARSCoV2_N:
+    description: >
+      Urine samples were collected and viral concentrations were quantified using
+      RT-qPCR targeting the N gene.
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: urine
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/mL
+    reference_event: symptom onset
+participants:
+- attributes:
```

Comment:
Need to create `race` and `ethnicity`: W = White; H = Hispanic; B = Black; O = Other; U = Unknown
.  Please check the scheme to see what are the current settings for `race` and `ethnicity`.

Reply:
May also include the `death` as an attribute.

Reply:
Good point! I updated Race and Ethnicity based on the schema. For Race_Ethnicity = H, the race is unknown, and the ethnicity is Hispanic. There is also BMI and diabetes information that can be added to subject attributes. Should I add this information to the .yaml file and update the schema as well?